src/proto/streams/counts.rs

```diff
@@ -47,7 +47,7 @@ impl Counts {
     }
 
     pub fn has_streams(&self) -> bool {
-        self.num_send_streams != 0 || self.num_recv_streams != 0 || self.num_reset_streams != 0
+        self.num_recv_streams != 0 || self.num_reset_streams != 0
```

Comment:
I think it was `num_reset_streams` that was supposed to be ignored? `num_send_streams` is important. Is that correct @seanmonstar?